ts/src/config.ts

```diff
@@ -125,6 +126,18 @@ export interface Config extends GoogleAuthOptions {
   // Duration of time profiles collected when using startLocal().
   // For testing with startLocal() only.
   localTimeDurationMillis?: number;
+
+  // When true, source map support will be disabled.
+  disableSourcemaps?: boolean;
```

Comment:
Nit: `disableSourceMaps`? "source maps" are two words, so I'd expect "maps" to be capitalized. Or are you following the naming convention used by the debugger modules?

Reply:
Done.